backend/http_settings.go

```diff
@@ -16,7 +16,7 @@ type HTTPSettings struct {
 	BasicAuthEnabled  bool
 	BasicAuthUser     string
 	BasicAuthPassword string
-	Headers           map[string]string
+	Headers           map[string][]string
```

Comment:
I would just go ahead and define this as a `http.Header`

Reply:
Good idea @andresmgot.

I have done that, and renamed from `Headers` to `Header`, as I broke API anyway (and this is now same as `net/http`.